#5382 Corrected Single.delay documentation

mithunsasidharan opened the pull request:
Corrected Single.delay documentation that says that success or error signals will be delayed by the specified amount, but in fact errors are not delayed, only successes.

mithunsasidharan committed:
#5382 Corrected Single.delay documentation 

Corrected Single.delay documentation that says that success or error signals will be delayed by the specified amount, but in fact errors are not delayed, only successes.

akarnokd commented on src/main/java/io/reactivex/Single.java:
```diff
@@ -1633,8 +1633,7 @@
     /**
-     * Delays the emission of the success or error signal from the current Single by
-     * the specified amount.
+     * Delays the emission of the success signal from the current Single by the specified amount.
```
Please update the `@param time` line as well

mithunsasidharan replied:
@akarnokd : Done. Could you review. Thanks

mithunsasidharan committed:
#5382 Corrected Single.delay documentation

Corrected Single.delay documentation that says that success or error signals will be delayed by the specified amount, but in fact errors are not delayed, only successes.
